python/setup.py.in

```diff
@@ -3,7 +3,11 @@ from distutils.command.build_ext import build_ext
 from distutils.sysconfig import customize_compiler
 import os
 
-os.environ["CC"] = "g++"
+if "CROSS_BUILD_ENVIRON" in os.environ and os.environ["CROSS_BUILD_ENVIRON"] == "y":
```

Comment:
please refer to my comment https://github.com/opencomputeproject/SAI/pull/1267#discussion_r663336272